mautic/mautic
# [UX] Improve the Custom headers tooltip to explain a use case

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

A user who clicks on the option without knowing it does not have any clue to help understand what the use of it is. The idea would be to improve the text of that tooltip to exemplify it in more detail.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Emails
3. Create new
4. Go to the Advanced tab
5. Hover your mouse over the Custom headers label
6. Read the new tooltip

![image](https://github.com/mautic/mautic/assets/116097999/713a122d-c2bd-4fc9-a419-eb55bc871580)
